lib/uploadcare/rails/active_record/mount_uploadcare_file.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'active_support/concern'
+
+module Uploadcare
+  module Rails
+    # A module containing ActiveRecord extension. Allows to use uploadcare file methods in Rails models
+    module ActiveRecord
+      extend ActiveSupport::Concern
+
+      UUID_REGEX = /\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b/.freeze
+
+      def build_uploadcare_file(attribute)
+        cdn_url = attributes[attribute.to_s].to_s
+        return nil if cdn_url.empty?
+
+        uuid = cdn_url.match(UUID_REGEX).to_s
```

Comment:
Can we move the logic of the UUID extraction to the gem? Can this logic be different for different CDNs?

Reply:
**Can we move the logic of the UUID extraction to the gem? **
We can move this logic to ruby-gem but is it ok if this logic is not used there?

**Can this logic be different for different CDNs?**
I think it can be a problem if a CDN-url includes several UUIDs.
@dmitry-mukhin, is it possible?

Reply:
yes, URLs can have several UUIDs.
e.g.:
- another file used for [watermarks](https://uploadcare.com/docs/transformations/image/overlay/), 
- part of a [filename](https://uploadcare.com/docs/delivery/cdn/#cdn-filename)